frontend/react/style-guide.md

```diff
@@ -1351,6 +1353,121 @@ const { isLoading, isError, error, isSuccess, data, status, fetchStatus } = useQ
   >- The **`fetchStatus`** gives information about the **`queryFn`**: Is it running or not?
 
 
+## Context
+
+### Introduction
+Context provides a way to share values like these between components without having to explicitly pass a prop through every level of the tree.
+>For advanced information on Context, check [here](https://tkdodo.eu/blog/practical-react-query).
```

Comment:
check link 👍🏻

Reply:
Fixed, I added a documentation that seemed good to me 😄